src/Classes/Item.lua

```diff
@@ -1161,6 +1161,10 @@ function ItemClass:BuildModListForSlotNum(baseList, slotNum)
 			flaskData[value.key] = value.value
 		end
 	elseif self.type == "Jewel" then
+		if string.find(self.name, "Grand Spectrum") then
```

Comment:
```suggestion
		if self.name:match("Grand Spectrum") then
```
Avoid accessing the global `string`

Reply:
Updated